libyul/AssemblyStack.cpp

```diff
@@ -260,11 +248,35 @@ std::pair<MachineAssemblyObject, MachineAssemblyObject> AssemblyStack::assembleW
 	if (subIndex.has_value())
 	{
 		evmasm::Assembly& runtimeAssembly = assembly.sub(*subIndex);
-		deployedObject.bytecode = make_shared<evmasm::LinkerObject>(runtimeAssembly.assemble());
-		deployedObject.assembly = runtimeAssembly.assemblyString();
+		return {make_shared<evmasm::Assembly>(assembly), make_shared<evmasm::Assembly>(runtimeAssembly)};
+	}
+
+	return {make_shared<evmasm::Assembly>(assembly), {}};
+}
+
+std::pair<MachineAssemblyObject, MachineAssemblyObject> AssemblyStack::assembleWithDeployed(optional<string_view> _deployName) const
+{
+	auto [creation, deployed] = assembleEVM(_deployName);
```

Comment:
These two read like booleans which throws me off when I see them in expressions. How about this?
```suggestion
	auto [creationAssembly, deployedAssembly] = assembleEVM(_deployName);
```

Reply:
Added.